instrumentation/SanitizerCoveragePCGUARD.so.cc

```diff
@@ -992,6 +1011,15 @@ bool ModuleSanitizerCoverage::InjectCoverage(Function &             F,
                   FixedVectorType::get(Int32PtrTy, elements);
               Value *x, *y;
 
+              if (!FunctionGuardArray) {
```

Comment:
@vanhauser-thc `scan-build` complained that `FunctionGuardArray` may be set to `null` here, maybe either fixing the root cause or exiting the build here could be a better solution (and at two other locations)

Reply:
these are false positives in scan-build. it is impossible that FunctionGuardArray is NULL, otherwise the function would not be called ... I think better than adding that if (!...) rather put a scan-build comment that this is a false positive? (sure there is a way to flag these)

Reply:
I prefer the if, the flow looks sane I think so there is a chance this might happen, and if not, it doesn't hurt